### ISSUE TITLE
wrong self link

### ISSUE THREAD

**Issue by BenjaminHorn** [digitalsadhu/loopback-component-jsonapi#49]
**http://localhost:3000/api/services** returns. The "self" link does not represent the item's id.
```javascript
{
  "data": [
    {
      "type": "services",
      "relationships": {
        "category": {
          "data": null
        }
      },
      "id": "1",
      "attributes": {
        "name": "Service 1"
      },
      "links": {
        "self": "http://localhost:3000/api/services/1"
      }
    },
    {
      "type": "services",
      "relationships": {
        "category": {
          "data": null
        }
      },
      "id": "2",
      "attributes": {
        "name": "Service 2"
      },
      "links": {
        "self": "http://localhost:3000/api/services/1"
      }
    }
  ],
  "links": {
    "self": "http://localhost:3000/api/services"
  }
}
```

**Comment by digitalsadhu** [comment]
Thanks for submitting the issue!
I've added a failing test case here: 2b4039ad319bb9b773ad4f26b2bd34597824d016

**Comment by digitalsadhu** [comment]
@listepo looks like this is in the serializer. Do you want to take a look?

**Comment by tsteuwer** [comment]
I'll try my hand at this one.

**Comment by digitalsadhu** [comment]
Fixed in #60
